### PR TITLE
drop subnet from ip address

### PR DESCRIPTION
**Pull request by daneharrigan**

### CODE REVIEW

File: dyno_driver.go [hunk 1]
```diff
@@ -88,13 +88,14 @@ func lookupLocalIP() (string, error) {
...
 	for _, addr := range addrs {
-		ip, ok := addr.(*net.IPNet)
-		if !ok && ip.IP.IsLoopback() {
+		ipnet, ok := addr.(*net.IPNet)
+		if !ok && ipnet.IP.IsLoopback() {
 			continue
 		}
 
-		if v4 := ip.IP.To4(); v4 != nil {
-			return v4.String(), nil
```
Comment (fabiokung) [inline dyno_driver.go]: According to the doc, this should not contain the subnet already: http://golang.org/pkg/net/#IP.String

**Comment by fabiokung** [comment]
1 minor comment, otherwise lgtm.

Can you please add a check to one of the tests? (or a new test)

**Comment by daneharrigan** [comment]
@fabiokung reverted `dyno_driver.go` and added another check in the socket status ftest

**Comment by fabiokung** [comment]
:+1:

**Pull request merged**